### PR TITLE
Separate Captain and F/O displays

### PR DESCRIPTION
**Pull request by tyler58546**
**Summary of Changes**

* Separated Captain and F/O PFD and ND
* FD and LS buttons no longer affect both displays
* ND filters no longer affect both displays
* Terrain radar buttons no longer affect both displays
* Turning on one display will no longer trigger a self test on the other display

**Screenshots (if necessary)**

image

**References**

N/A
**Additional context**

N/A

Discord username (if different from GitHub):

### CODE REVIEW

**Comment by Ares1378** [comment]
Tyler you are a bosss

**Comment by Benjozork** [comment]
Neat ! One little thing, remove changes to `layout.json` in your PR as this is handled by CI

**Comment by Pleasure0102** [comment]
Awesome job dude

**Comment by UncleClapton** [comment]
I haven't loaded this in to test, but is there any performance impact to doing this?

**Comment by NathanInnes** [comment]
Tested and works great!

**Comment by tyler58546** [comment]
> 
> 
> I haven't loaded this in to test, but is there any performance impact to doing this?

For me, each screen increases the average cpu time per frame by about 1ms

**Comment by NathanInnes** [comment]
Adding additional comment. I don't have much of a performance hit apart from a slightly increased CoherentGTDraw time.

**Comment by Pleasure0102** [comment]
tested, works extremely well! Thanks @tyler58546 needs merge 💯

**Comment by Pleasure0102** [comment]
not sure if bug for this fork, my ils info display seems to not turn on
check video here :
https://streamable.com/cvy9aw

**Comment by ThePilot916** [comment]
Nice!

**Comment by tyler58546** [comment]
> 
> 
> not sure if bug for this fork, my ils info display seems to not turn on
> check video here :
> https://streamable.com/cvy9aw
> 
> EDIT:
> checked latest master, so latest master the ILS info display works, so this is still a bug here

It works fine for me.

**Comment by Pleasure0102** [comment]
well can someone pull this and check ils info display on pfd ?

**Comment by lukecologne** [comment]
Works perfectly fine for me. The altimeter is still synced, but no issues with the LS buttons

**Comment by Pleasure0102** [comment]
weird... i mean you can clearly see in my video it does not work for me, and i have the same branch here. Everything else works

**Comment by Pleasure0102** [comment]
so again, redownloaded latest version from your repo and here is the result:
https://streamable.com/2yejb6

i mean you can clearly see in the video there is a problem not like i am making this up ;)

**Comment by lukecologne** [comment]
Ok, I tested it again, this time from cold and dark, and I can confirm @Pleasure0102's problem. It works fine when starting on the runway.

**Comment by Pleasure0102** [comment]
so i have spoken with @lukecologne it is a problem when starting cold and dark, on runway it works fine @tyler58546

**Comment by lukecologne** [comment]
Also looked into the altimeter setting, it looks like it is possible to use an index after the KOHLSMANN SETTING simvars. If no index is given, then it uses the value from index 1. This should make it possible to implement different altimeter settings, and stuff that doesn't use an index in the simvar just uses the captains setting, which should be accurate (an IRL a320 pilot should confirm that though). Separating the unit selector should be possible as well after looking at the model xml

**Comment by Ares1378** [comment]
None of the EFIS pb's work besides the FD pb's starting cold and dark. I have been told these work if you load in on the runway with engines running though.

**Comment by tareksabet** [comment]
So the P/R conflicts with Navigraph which makes the ND stuck on ILS mode but when I uninstalled it, it worked normally.
But, if you spawned the plane on a Gate the LS pb doesn't work but if spawned on the runway everything works.

**Comment by tyler58546** [comment]
Everything should work when starting cold and dark now.

**Comment by Pleasure0102** [comment]
> Everything should work when starting cold and dark now.

can confirm, cold and dark works now

**Comment by tyler58546** [comment]
PR now works with 1.9.3. This PR does not add any new files that were changed in the update.

**Comment by Pleasure0102** [comment]
thankfully !

**Comment by camilo625** [comment]
Name: Camilo625 #5575
Date of testing: Current date 30/09/2020
Version of the sim: 1.9.3.0
PR Tested: #934 

Testing technique: Spawn on gate and check displays on land and on air
Steps you've made to test the Pull Request:
1. Spawn on gate
2. Check displays while initializing systems
3. Takeoff
4. Check displays while flying

Overall Rating: Good
Conclusions of the testing: Seems to work perfectly, amazing work.

**Comment by starvingjavi** [comment]
Tried testing it in 1.9.3

The Capt and FO PFD are separate and work ok. Tested FD and ILS on both and it works separately without any noticeable frame loss.

The ND is stuck in the ILS mode for both the Capt and FO. Range is also not possible to change

Captura de pantalla 2020-09-30 172443

**Review by AdenFlorian — APPROVED** [review]
code looks fine, good to merge once testing passes

**Comment by lukecologne** [comment]
Merged into master for testing and works fine for me, except for the TERR ON ND modes, both buttons are activating the terrain display on the left ND and both button lights are synced.

**Comment by tareksabet** [comment]
everything works fine on my side, thank you

**Comment by lukecologne** [comment]
Alright, got a fix, it was a problem with the model XML. Will put it in a suggestion so you can commit easily. I'll also rename the TERRONND template to the A32NX convention.

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -13,4 +13,5 @@
 6. [EXTERIOR] Fixed issue in engine rotation animation that made it rotate too quickly - @lukecologne (lukecologne)
 7. [PFD] Remove code related to unrelevant aircraft - @1Revenger1 (Avery Black)
 8. [ECAM] Lower ECAM DOOR Page Colour Fix - @nathaninnes (Nathan Innes)
-9. [ND] Add DME distances, VOR/ADF needles and functioning ADF2 - @blitzcaster (bltzcstr)
\ No newline at end of file
+9. [ND] Add DME distances, VOR/ADF needles and functioning ADF2 - @blitzcaster (bltzcstr)
+1. [MISC] Separated Captian and F/O displays - @tyler58546 (tyler58546)
```
Comment (NathanInnes) [inline .github/CHANGELOG.md:17]: Change to current month

Comment (wpine215) [reply .github/CHANGELOG.md]: Fixed.

**Comment by ErikNL80** [comment]
Too bad performance is not taken into account when testing this... Maybe this could be added as part of the test routine?

**Pull request merged**